Automattic/wp-calypso
# Make sure isEligibleForCheckoutToChecklist returns false for the eCommerce plan

#### Changes proposed in this Pull Request

This PR updates `isEligibleForCheckoutToChecklist ` to make sure it return false when there is an eCommerce plan in the shopping cart. The rationale for this change is that users purchasing an eCommerce plan should be redirected to a dedicated checklist, not the default one.

#### Testing instructions

1. Purchase an eCommerce plan without choosing any segment in the first step
2. Pay for it
3. Confirm you got redirected to the correct eCommerce checklist
4. Purchase a different plan such as the Business plan
5. Pay for it
6. Confirm you got redirected to the WordPress.com checklist
